src/main/webapp/app/entities/dashboard/dashboard-card/dashboard-card-content.tsx

```diff
@@ -4,17 +4,19 @@ import MoreSmallListVert from '@spectrum-icons/workflow/MoreSmallListVert';
 import InfoOutline from '@spectrum-icons/workflow/InfoOutline';
 import { Translate } from 'react-jhipster';
 import { Redirect } from 'react-router-dom';
+import { hasAuthority } from 'app/shared/reducers/authentication';
 
 interface IDashboardCardContentProps {
   dashboardName: string;
   dashboardType: string;
   dashboardDescription: string;
   dashboardId: number;
   datasource: string;
+  account: any;
```

Comment:
create model for account. it can not be any

Reply:
done